content/en/docs/concepts/services-networking/service.md

```diff
@@ -184,6 +184,33 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Endpoint Slices (Alpha)
+Endpoint Slices are a new API resource designed to provide a more scalable
+alternative to Endpoints. Instead of all endpoints for a given service being
+stored in a single Endpoints resource, Endpoint Slices will distribute them
+across multiple resources. 
+
+An equivalent EndpointSlice resource to the Endpoints resource shown above would
+look fairly similar:
+
+```yaml
+apiVersion: discovery/v1alpha1
+kind: EndpointSlice
+metadata:
+  name: my-service.abcd
+subsets:
+  - endpoints:
+      - addresses: [192.0.2.42]
```

Comment:
Does this work without quoting?

Reply:
I thought so? At least ips work with a `-` array, not 100% sure here, but it was weird, so I cleaned it up, good catch.